src/main/java/org/cdlib/http/HttpClientFacadeImpl.java

```diff
@@ -43,14 +43,13 @@ public String post(String url, String post, int timeout, ContentType contentType
       httpPost.setEntity(requestEntity);
       HttpResponse response = httpClient.execute(httpPost);
       if (response.getStatusLine().getStatusCode() < 200 || response.getStatusLine().getStatusCode() > 399) {
-        LOGGER.error("Response status line = " + response.getStatusLine() + ", code = " + response.getStatusLine().getStatusCode() + ", from URL " + url + " post " + post);
```

Comment:
These logging statements have been removed because they are going to SYSOUT and creating noise picked up by IAS in the Tomcat systemd service. To restore this level of logging we need to take care of the Log4J 2 configuration.